### PR TITLE
Add note on PHP 8.1

### LINKED ISSUE #7023: WordPress is not fully compatible with PHP 8.0 or 8.1
Priority: Medium

## Issue Description

* Should any of the details from the Context section below be added to https://pantheon.io/docs/php-versions or https://pantheon.io/docs/wordpress-known-issues

### How will this impact users?

* This lets users know about possible issues before updating their site to 8.x

### Context

According to https://make.wordpress.org/core/2022/01/10/wordpress-5-9-and-php-8-0-8-1/, 
> PHP 8.1 was released on 25 November 2021. PHP 8.1 contains new features, performance improvements, deprecations, and backward compatibility breaks. For more information, see the PHP 8.1 release page, changelog, and migration guide.
> 
> WordPress is not fully compatible with PHP 8.0 or 8.1. All remaining known PHP 8.1 issues are deprecation notices.
> 
> Please note, a deprecation notice is not an error, but rather an indicator of where additional work is needed for compatibility before PHP 9 (i.e. when the notices become fatal errors). With a deprecation notice, the PHP code will continue to work and nothing is broken.

The link above includes "a breakdown of changes in WordPress Core that plugin and theme developers need to be aware of and accommodate in their code."

### PR DESCRIPTION
**Pull request by joa-pan**
Closes #7023 

## Summary

**[PHP versions](https://pantheon.io/docs/php-versions)** - Add compatibility info for PHP 8.1

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7234](https://pr-7234--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `46` | `46` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7234` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/35581/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/35581/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7234--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by CdrMarks** [comment]
@joa-pan I'd make the following changes:

1. Change the Alert Title to be "Compatibility Notes"
2. Change the Popover content to "See the Compatibility Notes section below".
3. Add the Popover to PHP version 8.0 in addition to 8.1
5. In the Alert body, separate the WordPress compatibility note out into its own line.  Add a link to https://make.wordpress.org/core/2022/01/10/wordpress-5-9-and-php-8-0-8-1/ and include, in your own words, "
Work is still ongoing and any open tickets in the post you linked are still relevant."

**Comment by joa-pan** [comment]
Thanks @CdrMarks. I updated the info based on your recommendations. I think we should stick with "Compatibility Note" as the admonition is  "Note" (just to be consistent), but I incorporated your other suggestions . Thanks a million for the review

**Comment by CdrMarks** [comment]
[Preview](https://pr-7234--pantheon-docs.my.pantheonfrontend.website/docs/php-versions)

**Comment by CdrMarks** [comment]
@joa-pan I think it's redundant to have any details in the popover.  Can you direct the reader to the Compatibility Considerations section.  

I'd recommend you use an Alert box so that the Compatibility Considerations stands out.

Can you also link 8.1 in the table to https://v81-php-info.pantheonsite.io/ ?

**Comment by whitneymeredith** [comment]
LGTM

File: source/content/php-versions.md [hunk 1]
```diff
@@ -55,6 +55,15 @@ End-of-life (**EOL**) versions are available on the platform but no longer under
 
 \* Sites that use this version of PHP will continue to serve pages, but new development cannot be done. The behavior of the development environment is undefined and not supported. To resume development on a site using a retired version of PHP, upgrade the PHP version on the development environment. 
 
+<Alert title="Note" type="info">
+
+PHP 8.1 contains new features, performance improvements, deprecations, and backward compatibility breaks and was released November 25, 2021.
```
Comment (CdrMarks) [inline source/content/php-versions.md]: I like this part and would have left it in place.

File: source/content/php-versions.md [hunk 2]
```diff
@@ -28,8 +28,8 @@ Changes made to the `pantheon.yml` file on a branch **are not** detected when cr
 
 | Version                                      | Recommended |  Status |
 | --------------------------------------------:|:-----------:| ------- |
-| 8.1 | ➖ | Available <Popover title="New Relic" content="New Relic is not supported in PHP 8.1." /> |
-| [8.0](https://v80-php-info.pantheonsite.io/) | ✅          | Active  |
+| [8.1](https://v81-php-info.pantheonsite.io/)| ➖ | Available <Popover title="Compatibility Note" content="WordPress is not fully compatible with PHP 8.1. New Relic is not supported in PHP 8.1." /> |
+| [8.0](https://v80-php-info.pantheonsite.io/) | ✅          | Active <Popover title="Compatibility Note" content="WordPress is not fully compatible with PHP 8.0." /> |
```
Comment (CdrMarks) [inline source/content/php-versions.md:32]: I believe the content here should direct the reader to the Compatibility Considerations section so we're not repeating ourselves.

**Review by CdrMarks — COMMENTED** [review]
@joa-pan @whitneymeredith 
I see this was merged, but feel there is still work to be done here.

File: source/content/php-versions.md (hunk 2 above)
Comment (joa-pan) [reply source/content/php-versions.md:32]: Thank you @CdrMarks. It's a good idea to direct users to that section. I'll open a new PR to add that change. Thanks a miilion

**Pull request merged**